_drafts/newsletter-002.md

```diff
@@ -42,6 +40,372 @@ _Discussions:
 If needed, a section can be split into subsections with a "------" delimiter.
 -->
 
+### Rust [Roguelike Toolkit][rl-toolkit] and [Roguelike Tutorial][rl-book]
+
+![Minimal pathfinding and FoV example](/assets/newsletter-002/rltk-rs-example-06.gif)
+
+[rltk_rs] by [@herberticus] is a Rust implementation of
+[C++ Roguelike Toolkit][RLTK] ([what is a "roguelike?"][rl]).
+
+It provides all the basic functionality one needs to write a roguelike game,
```

Comment:
```suggestion
It provides all the basic functionality one needs to write a roguelike game:
```

Reply:
This is supposed to be just the first item of the list. Maybe this should be rephrased somehow?  Hmm

Reply:
If that's the case, you can just leave the comma. Alternatively, you can try: